src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -101,11 +109,12 @@ def format_issue_table(diff_filename: str, issues: List[PullRequestIssue], toggl
 def get_issue_table_contents(issue_list: List[Dict[str, int]]) -> List[PullRequestIssue]:
     group_id_to_info = {}
     for issue in issue_list:
-        group_id = issue.pop("group_id")
-        group_id_to_info[group_id] = issue
+        group_id = issue["group_id"]
+        group_id_to_info[group_id] = {k: v for k, v in issue.items() if k != "group_id"}
```

Comment:
🤔 

do you need to do `k: v for k, v in issue.items()`?
seems weird to destructure it like that...
maybe `issue.filter().items()`?

Reply:
TIL dict filtering